src/constants.py

```diff
@@ -92,3 +93,7 @@
 # Timeout in seconds for Docker API client.
 API_CLIENT_TIMEOUT = 600
 MAX_WORKER_COUNT_FOR_PUSHING_IMAGES = 3
+
+PR_CREATION_DATA_HELPER_BUCKET = "pr-creation-data-helper"
```

Comment:
Please add a TODO to make this bucket account-specific, like `pr-creation-data-helper-<12_digit_account_id>`.

Reply:
Yes, will add and make the change in the upcoming PR along with this change: https://github.com/aws/deep-learning-containers/pull/3284#discussion_r1394687611